python/servo/package_commands.py

```diff
@@ -183,9 +182,29 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
             tar_path = '/'.join(dir_to_package.split('/')[:-1]) + '/'
             tar_path += datetime.utcnow().replace(microsecond=0).isoformat()
             tar_path += "-servo-tech-demo.tar.gz"
-            with tarfile.open(tar_path, "w:gz") as tar:
-                # arcname is to add by relative rather than absolute path
-                tar.add(dir_to_package, arcname='servo/')
+
+            # Archive deterministically
+            # See https://reproducible-builds.org/docs/archives/
+            archive_root_dir = 'servo'
+            with cd(dir_to_package):
```

Comment:
I think it would be a good idea to hoist this into a separate function to create a tarball, deterministically. I think `python/servo/command_base.py` is a good place for this.